### DISCUSSION TITLE
Installing bare metal @ Ubuntu 18.04: Setup reports error, but nothing in Logs

### DISCUSSION THREAD

**Discussion by revast** [Security-Onion-Solutions/securityonion#5668] [Unsupported Versions] [answered]
Hello, I have a Problem with installing, 

>Exact version. ex. 2.3.0

cat /etc/soversion
`2.3.70`
>Install source. Did you install from our Security Onion ISO image or did you perform a network installation?

network installation
>If network install, did you install from CentOS 7 or Ubuntu 18.04?

Ubuntu 18.04
>Install type. ex. eval, standalone, forward node, search node, etc

standalone
>Does so-status show all services running?

yes
> Do you get any failures when you run salt-call state.highstate from the node?

```
 Summary for local
--------------
Succeeded: 592 (changed=24)
Failed:      0
--------------
Total states run:     592
Total run time:   206.754 s

```

>Explain your issue. For example: Installation fails when I select this series of options...

Setup said it finished with errors, I should check the Logs. But No errors in Logs found..

cat /root/errors.log 
is empty

cat /root/so-setup.log
```
 Summary for local
--------------
Succeeded: 555 (changed=52)
Failed:      0
--------------
Total states run:     555
Total run time:   487.952 s
Errors detected during setup; skipping post-setup steps to allow for analysis of failures.
=========================================================================
Stopping mysql...
=========================================================================
so-mysql
so-mysql
```

cat /root/sosetup.log | grep error

```
 *** Restarting Salt to fix any SSL errors. ***
+log_level: error
+log_level_logfile: error
                  +log_level: error
                  +log_level_logfile: error

```

Oh, and `git clone https://github.com/Security-Onion-Solutions/securityonion` did not work on Ubuntu, 
one had to use `git clone https://github.com/Security-Onion-Solutions/securityonion.git`

Has anybody an Idea what is going on, or can I Ignore this Message?

**Comment by dougburks** [comment, marked as answer]
> Has anybody an Idea what is going on, or can I Ignore this Message?

If everything appears to be working correctly, then you can most likely ignore the message.

> Oh, and `git clone https://github.com/Security-Onion-Solutions/securityonion` did not work on Ubuntu,
> one had to use `git clone https://github.com/Security-Onion-Solutions/securityonion.git`

Strange, we've been using `git clone https://github.com/Security-Onion-Solutions/securityonion` for years and I don't think we've ever had any reports like this.
